src/v/coproc/event_listener.cc

```diff
@@ -99,17 +106,30 @@ ss::future<> event_listener::persist_actions(
               "Failed to register coprocessors with the wasm engine: {}",
               err);
             _offset = last_offset;
+            co_return;
+        }
+        for (const auto& id : enable_ids) {
```

Comment:
Looks like `script_id` is a `uint64_t` so it'll be more efficient to pass it by value here.